apps/ledger-live-mobile/src/screens/Manager/Device/DeviceOptionRow.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Flex, Icons, Link, Text } from "@ledgerhq/native-ui";
+import { IconType } from "@ledgerhq/native-ui/components/Icon/type";
+
+type Props = {
+  Icon: IconType;
+  iconSize?: number;
+  label: string;
+  linkLabel: string;
+  onPress?: () => void;
+  right?: React.ReactNode;
```

Comment:
for a component, i would have put a capitalized name, like `Right`, but i'm an annoying person 😈

Reply:
It's not a component, it's a `ReactNode`, like `children` for instance, so no capitalisation, right?